stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/PhoneNumberState.kt

```diff
@@ -0,0 +1,18 @@
+package com.stripe.android.uicore.elements
+
+import androidx.annotation.RestrictTo
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
```

Comment:
This wasn't restricted before, but is now. We need to be extra sure this is ok.

Reply:
The only place it was used was in `AddressType`, which is already restricted, so it's fine to hide it too